doc/sphinx/src/interface/state.rst

```diff
@@ -112,6 +112,13 @@ several useful features and functions.
   deletgates to the ``std::function`` member ``PostStepDiagnosticsMesh``
   if set (defaults to ``nullptr`` an therefore a no-op) to print
   diagnostics after the time-integration advance
+- ``void UserWorkBeforeLoopMesh(Mesh *, ParameterInput *pin, SimTime
```

Comment:
Before reading further, this naming (i.e., `UserWorkBeforeLoopMesh`) made me think that this was going to be called before every `Step()` cycle.  Does anybody else see this as a potential confusion? How about `UserInitFinalize` or something more indicative of its utility?

Reply:
I would find `UserInitFinalize` to be more confusing personally... 

I *think* the current name is unambiguous, as there's already a `PreStepUserWorkInLoop` and a `PostStepUserWorkInLoop`, which is work done per step. And the current name fits more in that scheme. But I don't feel strongly.